tcav/model_test.py

```diff
@@ -0,0 +1,120 @@
+"""
+Copyright 2019 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import tensorflow as tf
+from tensorflow.python.framework import graph_util
+from tensorflow.python.platform import googletest
+from tcav.model import ModelWrapper
+
+
+class ModelTest_model(ModelWrapper):
+  """A mock model of model class for ModelTest class."""
+
+  def __init__(self, model_path=None, node_dict=None):
+    super(ModelTest_model, self).__init__(
+        model_path=model_path, node_dict=node_dict)
+
+  def label_to_id(self, target_class):
+    return 0
```

Comment:
can we have our base class implement a naive version of this for simple models, with others needing to override it? can just return label as id and vice versa (forcing to numbers if necessary)?

Reply:
If user pass a string 'zebra', forcing it into a number in label_to_id might not make sense. I am defaulting label_to_id to return 0 with logging warnings. I am also defaulting id_to_label to cast idx into a string. id_to_label is not used for now though. Might be useful to keep it around.